charts/keycloak-operator/values.yaml

```diff
@@ -0,0 +1,283 @@
+    strictBackchannel: true
+    # -- The hostname for accessing the administration console.
+    admin: ~
+    # -- Set the base URL for accessing the administration console.
+    adminUrl: ~
+
+  unsupported:
+    # -- You can configure values that will be merged with the one configured by default by the operator.
+    podTemplate: ~
+      #    metadata:
+      #      labels:
+      #        foo: "bar"
+      #    spec:
+      #      containers:
+      #        - volumeMounts:
+      #            - name: test-volume
+      #              mountPath: /mnt/test
+      #          command: [ "/bin/bash", "-c", "cat /mnt/test/test.txt && /opt/keycloak/bin/kc.sh start-dev" ]
+      #      volumes:
+      #        - name: test-volume
+      #          secret:
+      #            secretName: keycloak-podtemplate-secret
+
+  ingress:
+    # -- The deployment is, by default, exposed through a basic ingress.
+    enabled: true
+
+  image:
+    # -- Overrides the operator.keycloakImage.image value whose default is quay.io/keycloak/keycloak
+    repository: ""
+    # -- Overrides the operator.keycloakImage.tag value whose default is the chart appVersion.
+    tag: ""
+
+  # -- Secret(s) that might be used when pulling an image from a private container image registry or repository.
+  imagePullSecrets: ~
+  # - name: secret-name
+
+  # -- Configuration of the Keycloak server expressed as a keys and values that can be either direct values or references to secrets.
+  additionalOptions: ~
+  # - name: string
+  #   value: string
+  #   secret:
+  #     optional: bol
+  #     key: string
+  #     name: string
+
+  db:
+    # -- The reference to a secret holding the password of the database user.
+    passwordSecret: ~
+    # optional: bool
+    # key: string
+    # name: string
+    # -- The reference to a secret holding the username of the database user.
+    usernameSecret: ~
+    # optional: bool
+    # key: string
+    # name: string
+    # -- Sets the port of the default JDBC URL of the chosen vendor. If the `url` option is set, this option is ignored.
+    port: ~
+    # -- The database schema to be used.
+    schema: ~
+    # -- Sets the hostname of the default JDBC URL of the chosen vendor. If the `url` option is set, this option is ignored.
+    host:
+    # -- The full database JDBC URL. If not provided, a default URL is set based on the selected database vendor.
+    url:
+    # -- The initial size of the connection pool.
+    poolInitialSize: ~
+    # -- The maximum size of the connection pool.
+    poolMaxSize: ~
+    # -- The database vendor.
+    vendor: ~
+    # -- Sets the database name of the default JDBC URL of the chosen vendor. If the `url` option is set, this option is ignored.
+    database: ~
+    # -- The minimal size of the connection pool.
+    poolMinSize: ~
+  features:
+    # -- Disabled Keycloak features
```

Comment:
It might be a good idea to add a link to the upstream docs here? Or is everything related to Keycloak configuration expected to be known by everyone deploying this service?

Reply:
I added a link in 96b6799a8537d05766a343bc2f68667465647a8d but i also assume some familiarity with keycloak and it's operator. I'm assuming this will hold true until the operator is more mature and reaches the auto pilot capability level.